recipes/clang/activate-clang++.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
```

Comment:
`#!/usr/bin/env bash` is better than `/bin/bash`, IMO, but this is not blocking.

Reply:
These scripts are sourced by conda, so it doesn't matter.